HW1_2.cpp

```diff
@@ -0,0 +1,133 @@
+#include <string>
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+
+//This program is aimed to solve Question 2 in Homework 1,so it's actually a simplified version of global alignment,which means some parameters it uses are fixed
+
+//return the maxium of three numbers
+int max3(int a, int b, int c)
+{
+	int max1 = a > b ? a : b;
+	int max2 = c > max1 ? c : max1;
+	return max2;
+}
+
+
+int main()
+{
+	ifstream infile;
+	ofstream outfile;
+	string seq1="";
+	string seq2="";
+
+	//reading sequence
+	infile.open("D://LongestCommonSeq.txt");
+	getline(infile, seq1);
+	getline(infile, seq2);
+	//output the length of sequence
+	if (seq1 != "" && seq2 != "")
+		cout << "Read the file successfully" << endl << "sequence length: " << seq1.length() << "\t" << seq2.length() << endl;
+	else
+		return 0;
+
+	int height = seq1.length() + 1;
+	int width = seq2.length() + 1;
+	int **data;
```

Comment:
In Cpp, my personal experience is to use vector<vector<int> > matrix to present one int type matrix instead of using the pointer format in C~ 
Songpeng